samples/audioProfile.js

```diff
@@ -14,43 +14,38 @@
  */
 
 'use strict';
+function synthesizeSpeechAsync(request = {}) {
+  // Imports the Google Cloud client library
+  const textToSpeech = require('@google-cloud/text-to-speech');
+  // Creates a client
+  const client = new textToSpeech.TextToSpeechClient();
+  return new Promise((resolve, reject) => {
```

Comment:
Once this is changed to an async function, you can just `await` the result of `client.synthesizeSpeech`

Reply:
used with original function.